// @selmathistckt/common/build/events/subjects.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Subjects = void 0;
var Subjects;
(function (Subjects) {
    Subjects["TicketCreated"] = "ticket:created";
    Subjects["TicketUpdated"] = "ticket:updated";
    Subjects["OrderCreated"] = "order:created";
    Subjects["OrderCancelled"] = "order:cancelled";
    Subjects["ExpirationComplete"] = "expiration:complete";
    Subjects["PaymentCreated"] = "payment:created";
    Subjects["BookingCreated"] = "booking:created";
    Subjects["BookingCancelled"] = "booking:cancelled";
    Subjects["BookingFailed"] = "booking:failed";
    Subjects["BookingCompleted"] = "booking:completed";
    Subjects["BusCreated"] = "bus:created";
    Subjects["BusUpdated"] = "bus:updated";
    Subjects["UserRegistered"] = "user:registered";
    Subjects["UserUpdated"] = "user:updated";
    Subjects["BusTicketCreated"] = "ticket:created";
    Subjects["BusTicketUpdated"] = "ticket:updated";
})(Subjects || (exports.Subjects = Subjects = {}));
